### PR TITLE
automation of building the workitem repository

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

**Comment by tsurdilo** [comment]
Jenkins retest please

File: archive-workitem/src/main/java/org/jbpm/process/workitem/archive/ArchiveWorkItemHandler.java [hunk 1]
```diff
@@ -36,11 +36,13 @@
...
         displayName = "Archive",
         defaultHandler = "mvel: new org.jbpm.process.workitem.archive.ArchiveWorkItemHandler()",
+        documentation = "${artifactId}/index.html",
         parameters = {
                 @WidParameter(name = "Archive"),
                 @WidParameter(name = "Files")
         },
         mavenDepends = {
+                @WidMavenDepends(group = "${groupId}", artifact = "${artifactId}", version = "${version}"),
                 @WidMavenDepends(group = "org.apache.commons", artifact = "commons-compress", version = "1.12")
```
Comment (mswiderski) [inline archive-workitem/src/main/java/org/jbpm/process/workitem/archive/ArchiveWorkItemHandler.java]: I don't think the org.apache.commons... is needed here as it is already a compile time dependency of archive-workitem module so will already be included.

File: dropbox-workitem/src/main/java/org/jbpm/process/workitem/dropbox/DownloadFileWorkitemHandler.java [hunk 2]
```diff
@@ -39,13 +39,15 @@
...
+        documentation = "${artifactId}/index.html",
         parameters = {
                 @WidParameter(name = "DocumentPath")
         },
         results = {
                 @WidResult(name = "Document")
         },
         mavenDepends = {
+                @WidMavenDepends(group = "${groupId}", artifact = "${artifactId}", version = "${version}"),
                 @WidMavenDepends(group = "com.dropbox.core", artifact = "dropbox-core-sdk", version = "3.0.5")
```
Comment (mswiderski) [inline dropbox-workitem/src/main/java/org/jbpm/process/workitem/dropbox/DownloadFileWorkitemHandler.java]: same here ... dropbox is dependency of dropbox-workitem and thus will be on classpath when this workitem module is added as dependency

**Comment by tsurdilo** [comment]
@mswiderski updated all handlers. thanks!

**Comment by tsurdilo** [comment]
jenkins seems to not be building with master jbpm -- its not including https://github.com/kiegroup/jbpm/commit/e3bc887dc43036cac2e71d3de6f6c10e1418bbbe for example...weird

**Comment by tsurdilo** [comment]
Jenkins execute full downstream build

**Pull request merged**